src/lib/booleans.js

```diff
@@ -0,0 +1,121 @@
+function negate(a) {
+  if (a === true) {
+    return false;
+  }
+  return true;
+}
+
+function both(a, b) {
+  if (a && b === true) {
+    return true;
+  }
+  return false;
```

Comment:
You can rewrite this function so it will return true based on a single condition:
```
  if (a && b) {
    return true;
  }
  return false;
```
Going a bit further you could turn it into a simple ternary operator:
```
return (a && b) ? true : false;
```
Which as it happens my editor simplified to just this!
```
  return (a && b);
```